src/components/SubmitTokenInfo/ImgUpload.module.scss

```diff
@@ -0,0 +1,69 @@
+.container {
+  font-size: 14px;
+}
+
+.label {
+  display: flex;
+  align-items: center;
+  color: #666;
+  font-size: 14px;
+  margin-bottom: 12px;
+}
+
+.labelRightAddon {
+  display: flex;
+  align-self: center;
+  margin-left: 4px;
+}
+
+.input {
+  display: none;
+}
+
+.inputWrapper {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 112px;
+  height: 112px;
+  border-radius: 4px;
+  border: 1px solid #e5e5e5;
+  background: white;
+  user-select: none;
+}
+
+.uploadIconWrapper {
+  cursor: pointer;
+}
+
+.uploadIcon {
+  width: 24px;
+  height: 24px;
+}
+
+.deleteIcon {
+  position: absolute;
+  top: 0;
+  right: 0;
+  border: none;
+  cursor: pointer;
+  background: rgb(0 0 0 / 50%);
+
+  img {
+    width: 16px;
+    height: 16px;
+  }
+}
+
+.uploadedIcon {
+  width: 112px;
+  height: 112px;
+  object-fit: scale-down;
```

Comment:
The reason of using `scale-down` instead of `cover`?

Reply:
Not a specific reason, this issue may be discussed here.

I though it would be better to show the full content of uploaded image rather than covering the component with uploaded image when the `width` and `height` are different.

cover: 

<img width="200" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/3870972/81eb74d0-40f7-445e-a4a9-ada252fb4f29">

scale down:

<img width="200" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/3870972/f136417e-2cfb-4eec-a535-ff91f27142f7">

But usually we can assume that the uploaded token icon is a sqaire image. In that case, the two types of apperance are similar.

which style do we prefer ? 
cc: @Sven-TBD

Reply:
I prefer to see the full logo